index.html

```diff
@@ -0,0 +1,579 @@
+<li><img class="solar" src="images/inflation-5.jpg" alt="big-bang"></li>
+<li><b>STILL NOT ENOUGH TIME</b> has not been enough time to carry coffee cups from place to place before they get cold. But if all the coffee cups were somehow filled from a single coffee pot, all at the same time, that might explain their even temperature.</li>
+</ul>
+        </div>
+        <div class="inflation-illustration-b">
+            <ul>
+                <li><img class="solar" src="images/inflation-2.jpg" alt="big-bang"></li>
+                <li><b>NOT ENOUGH TIME</b> The universe is not old enough for light to have traveled the vast distance from one side of the universe to the other, and there has not been enough time for scattered patches of hot and cold to mix into an even temperature.</li>
+            <li><img class="solar" src="images/inflation-6.jpg" alt="big-bang"></li>
+            <li><b>INFLATION</b> this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field eventually grew into the large-scale structures of the universe.</li>
+            </ul>
+        </div>
+        <div class="inflation-illustration-c">
+            <ul>
+                <li><img class="solar-b" src="images/inflation-3.jpg" alt="big-bang"></li>
+            <li><b>DISTANT COFFEE</b> At a smaller scale, imagine using a telescope to look a mile in one direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and how much it has cooled.</li>
+        <li><img class="solar-b" src="images/inflation-7.jpg" alt="big-bang"></li>    
+    <li><b>FLUCTUATION</b> have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.</li>    
+    </ul>
+        </div>
+        <div class="inflation-illustration-d">
+            <ul>
+                <li><img class="solar-b" src="images/inflation-4.jpg" alt="big-bang"></li>
+                <li><b>COFFEE EVERYWHERE</b> Now turn around and look a mile in the other direction. You see a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar cup in every direction, you might want to look for another explanation.</li>
+            <li><img class="solar-b" src="images/inflation-8.jpg" alt="big-bang"></li>
+            <li><b>EXPANSION</b> to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.</li>
+            </ul>
+        </div>
+    </div>
+    </article>
```

Comment:
- [ ] Please the theory of inflation section on your page should be arranged properly